pkg/config/config.go

```diff
@@ -112,16 +114,27 @@ type SecurityCapabilityLists struct {
 
 // ParseFile parses config from a file.
 func ParseFile(path string) (Configuration, error) {
-	configBox := packr.New("Config", "../../examples")
-	var rawBytes []byte
 	var err error
+	var rawBytes []byte
+	configBox := packr.New("Config", "../../examples")
 	if path == "" {
 		rawBytes, err = configBox.Find("config.yaml")
+		if err != nil {
+			return Configuration{}, err
+		}
+	} else if strings.HasPrefix(path, "https://") || strings.HasPrefix(path, "http://") {
+		//path is a url
+		response, err := http.Get(path)
+		if err != nil {
+			return Configuration{}, err
+		}
+		rawBytes, err = ioutil.ReadAll(response.Body)
 	} else {
+		//path is local
 		rawBytes, err = ioutil.ReadFile(path)
-	}
-	if err != nil {
-		return Configuration{}, err
+		if err != nil {
```

Comment:
This still has to go outside the `else` statement, or the `err` on line 131 won't get caught